docs/getting_started/creating_an_admin.rst

```diff
@@ -61,6 +61,12 @@ two very simple ``Post`` and ``Tag`` entities will be used::
     class Category
     {
         // ...
+       /**
+         * @ORM\Id
+         * @ORM\Column(type="integer")
+         * @ORM\GeneratedValue(strategy="AUTO")
+         */
+        protected $id;
```

Comment:
Why `protected`?

Reply:
Isn't the point of `// ...` data people should already have in their entity or in this case this `$id`.

Reply:
But that would mean that the new user didn't even bother to read Symfony/Doctrine docs, I don' t think we should add this, cc @sonata-project/contributors?

Reply:
I am against this change